src/components/user/uc/registrationPin.facade.js

```diff
@@ -0,0 +1,23 @@
+const registrationPinUc = require('./registrationPin.uc');
+
+class RegistrationPinFacade {
+	constructor(app) {
+		this.app = app;
+	}
+
+	async getRegistrationPinsForUser(email) {
+		return registrationPinUc.getRegistrationPinsForUser(email);
```

Comment:
getRegistrationPinsForUser and deleteRegistrationPins should exposed in this PR, missing permissions and so on